pkg/infra/usagestats/types.go

```diff
@@ -0,0 +1,10 @@
+package usagestats
+
+type ConcurrentUsersStats struct {
+	BucketLe3   int32
```

Comment:
What does "Le" stand for, little endian? If it's an initialism, it should be written in consistent case, to be comply with standard Go style.